russian/sleep/zefir.md

```diff
@@ -0,0 +1,7 @@
+Вы просыпаетесь и понимаете что всё ещё находитесь в комнате, стены которой сделаны из зефира. Стены очень приятно пахнут. Интересно, какие они на вкус?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.